### PR TITLE
Doc Action URL CLI Flag

### PR DESCRIPTION
**Pull request by dubeejw**
Document how to get an action and web action URL.

### CODE REVIEW

**Comment by dubeejw** [comment]
@bjustin-ibm, please review.

File: docs/actions.md [hunk 1]
```diff
@@ -271,6 +271,19 @@ Rather than pass all the parameters to an action every time, you can bind certai
...
   ```
 
+### Getting an action URL
+
+An action can be invoked through the REST interface via an HTTPS request. To get an action URL, execute the following command:
+
+```
+$ wsk action get actionName --url
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/actionName
```
Comment (bjustin-ibm) [inline docs/actions.md]: Is this consistent with how we show URLs in other parts of the documentation?

File: docs/actions.md [hunk 2]
```diff
@@ -271,6 +271,19 @@ Rather than pass all the parameters to an action every time, you can bind certai
...
+```
+
+**Note:** Authentication must be provided when invoking an action via HTTPS request.
```
Comment (rabbah) [inline docs/actions.md]: perhaps add a curl example?

**Comment by dubeejw** [comment]
Added link to REST interface doc.

**Comment by rabbah** [comment]
Thanks.

**Pull request merged**